service/service.go

```diff
@@ -319,6 +319,15 @@ func (s *service) probe(ctx context.Context, clusterConfig *IsilonClusterConfig)
 		if err := s.nodeProbe(ctx, clusterConfig); err != nil {
 			return err
 		}
+	} else if strings.EqualFold(s.mode, "") {
+		log.Warn("Service mode not set, attempting both controller and node probe")
+		controllerErr := s.controllerProbe(ctx, clusterConfig)
```

Comment:
fail if there is a controllerErr; no need to do nodeProbe()

Reply:
Addressed